src/elektra/redshift.ini

```diff
@@ -0,0 +1,148 @@
+; Elektra spec file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
```

Comment:
Also give a short intro what spec files are and link to https://www.libelektra.org/tutorials/writing-a-specification-for-your-configuration

Reply:
Thanks, done.